Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+Because of this the team members must be in the mindset to analyze each part of the project and identify as many problem areas that need answers as possible. 
+This means being specific on how we are going to accomplish our goals.
+We want our team asking the question:"what material, what algorithm, with what method will we be accomplishing this?"
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+These are skills that benefit RIT SPEX. 
+
+\section{Implementation}
+\label{sec:implementation}
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress.
+The rover project lends itself well to modularity.
+This will allow us to set up the project in a way that best matches the skillset of the team we have. 
+The team would take this information, refine it and put it into the technical report along with construction. 
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+The deliverable includes the technical report, rover software, and rover hardware.
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester.
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference. 
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+The project is intended to last 2 semesters with the deliverable of a mock rover and a technical report.   
+
+\subsubsection{Technical Report}
+\label{techreport}
+The technical report of the rover is a design document complete with the hardware, software, and electrical components of the rover. 
+This report is important as it allows for a future team to easily learn from the past team's experience. 
+This Technical Report will be hosted on GitHub. 
+This will also include the results of the rover tests and the teams recommendations based on them. 
+
+\subsubsection{Rover Software}
+\label{roversoftware}
+The rover software has to be written and hosted on GitHub. 
+This will include rover features such as navigation, control, and power regulation. 
+The rover need to be controlled so software must be made to do so. 
+This means the rover electronics must be integrated with a control application. 
+The rover will need to distribute power to the motors in the wheels so software will need to be made to regulate such power. 
+There will also need to be software for the autonomous or partially autonomous features on the rover. 
+
+\subsubsection{Rover Hardware}
+\label{roverhardware}
+The rover will need to be constructed and tested. Because of this a physically constructed rover is needed. This will be necessary for ImagineRIT for cool factor. This includes the rover body, mast, suspension, and wheels. 
+
+\begin{figure}[hb!]
```

Comment:
Figure is included, but doesn't appear to be referenced anywhere (especially not on the page it is currently on)